flutter/lib/src/default_integrations.dart

```diff
@@ -195,14 +196,87 @@ class _LoadContextsIntegrationEventProcessor extends EventProcessor {
         event = event.copyWith(contexts: eventContexts);
       }
 
+      final tagsMap = infos['tags'];
+      if (tagsMap != null) {
```

Comment:
```suggestion
      if (tagsMap?.isNotEmpty ?? false) {
```

That would spare some code to be run, same for other fields.